### DISCUSSION TITLE
Hacktoberfest is coming soon!

### DISCUSSION THREAD

**Discussion by cclauss** [TheAlgorithms/Python#8581] [Team Posts]
https://hacktoberfest.digitalocean.com

Are there things that we should be doing to prepare for the onslaught??

**Comment by cclauss** [comment]
https://github.com/TheAlgorithms/Python/issues/2510#issuecomment-713342893

#2510, #3609

**Comment by poyea** [comment]
Let's try to shorten the current PR queue a bit in the following 2 weeks?

**Comment by putul03** [comment]
Hi @poyea @darkstar @bit-man @spazm and the team! I understand you have a lot on your plate with many open pull requests. If you could take a moment to review my PR - Added Python Implementation of Suffix Arrays and LCP Arrays #12171, it would be greatly appreciated. I'm excited to contribute to the project and welcome any feedback you may have. Thank you!
